cooltools/api/expected.py

```diff
@@ -261,16 +259,13 @@ def _make_diag_table(n_bins, bad_locs):
     return diags.astype(int)
 
 
-def make_diag_tables(
-    clr, regions, regions2=None, clr_weight_name="weight", bad_bins=None
-):
+def make_diag_tables(clr, regions, regions2=None, clr_weight_name="weight"):
     """
     For every support region infer diagonals that intersect this region
     and calculate the size of these intersections in pixels, both "total" and
     "n_valid", where "n_valid" does not include "bad" bins into counting.
```

Comment:
For every region infer diagonals that intersect this region
    and calculate the size of these intersections in pixels, both "total" and
    "n_valid", where "n_valid" does not count "bad" pixels.